build.sbt

```diff
@@ -73,7 +74,7 @@ libraryDependencies ++= Seq(
     "net.logstash.logback" % "logstash-logback-encoder" % "4.11",
     "com.fasterxml.jackson.dataformat" % "jackson-dataformat-cbor" % "2.9.2",
     "com.google.guava" % "guava" % "25.0-jre", //-- added explicitly - snyk report avoid logback vulnerability
-    "com.fasterxml.jackson.core" % "jackson-databind" % jacksonVersion, //added explicitly to avoid snyk vulnerability
+    "com.fasterxml.jackson.core" % "jackson-databind" % (jacksonVersion+".1"), //added explicitly to avoid snyk vulnerability
```

Comment:
the other jackson libs didn't get the bump, only this one.  Doing it this way will force it to break once the next bump happens at compile time, which will trigger the next person to update it here.  If I hard code the version it may get missed and only changed for the others.